src/app/Instance/CreateInstance/machines/createInstanceMachine.ts

```diff
@@ -269,32 +282,42 @@ const createInstanceMachine = createMachine(
       }),
       setErrorHandler: assign((context, { method, parameters }) => {
```

Comment:
Any reason not to rename this `_context` too (for consistency)?

Reply:
I only added the trailing underscore where `context` is not used to silence the `no-unused-vars` lint rule. (I've tried to remove it entirely too but it seems `assign` can't live without it).

Here the context is used a couple of lines below so the underscore is not necessary.

This same answer is valid for other places where I didn't rename it.